rosdep/python.yaml

```diff
@@ -4291,29 +4291,13 @@ python-sip:
     slackpkg:
       packages: [sip]
   ubuntu:
-    artful: [python-sip-dev]
+    '*': [python-sip-dev]
     artful_python3: [python3-sip-dev]
```

Comment:
If I understand @tfoote's answer, the `_python3` keys on EOL platforms may be removed. Looks like the EOL ones are

* `artful_python3`
* `trust_python3`
* `wily_python3`
* `yakkety_python3`
* `zesty_python3`

Reply:
Thank, I removed these.